pkg/operations/kubernetesupgrade/upgradeagentnode.go

```diff
@@ -19,8 +19,9 @@ import (
 )
 
 const (
-	interval = time.Second * 1
-	retry    = time.Second * 5
+	interval           = time.Second * 1
+	retry              = time.Second * 5
+	cordonDrainTimeout = time.Minute * 10
```

Comment:
why 10 minutes? if scale is 1 hour

Reply:
This is just a placeholder to demonstrate the surface area that needs changing. No concrete requirement yet.